js/src/wrappers/anthropic.ts

```diff
@@ -1,13 +1,36 @@
-import Anthropic from "@anthropic-ai/sdk";
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
If we want to enforce this rule, we should add eslint to the build and have it start failing.